src/LanguageLine.php

```diff
@@ -53,7 +53,11 @@ public static function getCacheKey(string $group, string $locale): string
      */
     public function getTranslation(string $locale): string
     {
-        return $this->text[$locale] ?? $this->text[config('app.fallback_locale')];
+        if(isset($this->text[$locale]) && !is_array($this->text[$locale]))
```

Comment:
I don't like to have if conditions without curly brackets as [this can lead to some serious problems](https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/)